components/docs-chef-io/content/automate/supermarket_integration_with_automate.md

```diff
@@ -0,0 +1,185 @@
+
+1. SSH into to your ec2 instance and download the Chef Automate command-line tool from the current release channel.
+
+    ```cd
+    curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
+    ```
+
+1. Download the airgapped installation bundle of the latest automate version to an internet-connected machine using the following command:
+
+    ```bash
+    curl https://packages.chef.io/airgap_bundle/current/automate/latest.aib -o </path/to/airgap-install-bundle>
+    ```
+
+    Or download the airgapped bundle of a specific version of automate using the command below:
+
+    ```bash
+    curl https://packages.chef.io/airgap_bundle/current/automate/<version>.aib -o </path/to/airgap-install-bundle>
+
+    ```
+
+1. Deploy the Airgap Installation Bundle using the following command:
+
+    ```bash
+    chef-automate deploy --airgap-bundle automate-<version>.aib --product automate --product infra-server
+    ```
+
+1. Check the status of all the components using the following command:
+
+    ```bash
+    chef-automate status
+    ```
+
+1. Check the logs using the following command:
+
+    ```bash
+    journalctl -u chef-automate -f
+    ```
+
+1. Create a user using the following command:
+
+    ```bash
+    chef-server-ctl command
+    ```
+
+    N.B. For more help on how to create user using the `chef-server-ctl` utility, refer this [documentation](https://docs.chef.io/server/ctl_chef_server/#user-create)
+
+## Register Supermarket with Automate Embedded Chef Identity
+
+When you install Chef Automate, it bundles the Chef-Server OC-ID component as an Oauth provider. Users can use the Oauth provider to log in to another application (e.g. Supermarket) using their Chef-Server credentials. Follow the steps below to register the applications to use OC-ID as a medium to log in to the respective applications. Once you finish the registration, you will be authorized to use Chef-Server login credentials to login to the application.
+
+1. Create a file to list down the details of the application you want to register with OC-ID. In the file `ocid-apps.toml`, mention the application's **name** and the **redirect_uri**. The content of the created file should be in the following format:
+
+    ```cd
+    [ocid.v1.sys.ocid.oauth_application_config]
+        [[ocid.v1.sys.ocid.oauth_application_config.oauth_applications]]
+            name = ""
+            redirect_uri = ""
+    ```
+
+    Update the URL of the supermarket website in the `redirect_uri` as per the actual supermarket URL. Refer to the code below:
```

Comment:
URL of supermarket generally looks like - **https://example-supermarket.com**, but here we want user to set **redirect_uri** as `<supermarket-url>/auth/chef_oauth2/callback`. Do we need to explicitly mention that here?

Reply:
@dishanktiwari2501 ☝🏽

Reply:
@dishanktiwari2501 can we address this comment?